docs/src/man/basics.md

```diff
@@ -1510,32 +1510,96 @@ julia> german[Not(5), r"S"]
 
 ## Basic Usage of Transformation Functions
 
-In DataFrames.jl we have five functions that we can be used to perform transformations of columns
-of a data frame:
-
-- `combine`: creates a new data frame populated with columns that are results of transformation
-  applied to the source data frame columns, potentially combining its rows;
-- `select`: creates a new data frame that has the same number of rows as the source data frame
-  populated with columns that are results of transformations applied to the source data frame
-  columns;
-- `select!`: the same as `select` but updates the passed data frame in place;
-- `transform`: the same as `select` but keeps the columns that were already present in the data frame
-  (note though that these columns can be potentially modified by the transformation passed to `transform`);
-- `transform!`: the same as `transform` but updates the passed data frame in place.
-
-The fundamental ways to specify a transformation are:
-
-- `source_column => transformation => target_column_name`;
-  In this scenario the `source_column` is passed as an argument to `transformation` function
-  and stored in `target_column_name` column.
-- `source_column => transformation`;
-  In this scenario we apply the transformation function to `source_column` and the
-  target column names is automatically generated.
-- `source_column => target_column_name` renames the `source_column` to `target_column_name`.
-- `source_column` just keep the source column as is in the result without any transformation;
-
-These rules are typically called transformation mini-language.
-
+In DataFrames.jl there are seven functions that can be used to transform data frame columns:
```

Comment:
It seems to me that `subset` is not transforming columns of a data frame. Maybe use some more general name for the operation if you want to keep `subset` in the list? (keeping in mind that you do not list `filter` here)